common/men_at_arms_types/00_maa_types.txt

```diff
@@ -125,7 +130,36 @@ pikemen_unit = {
 	ai_quality = { value = culture_ai_weight_pikemen }
 	icon = pikemen
 }
+###WARCRAFT###
+caster_unit = {
+	type = caster
+	
+	damage = 30
+	toughness = 9
+	pursuit = 11
+	screen = 0
+	
+	terrain_bonus = {
+		plains = { damage = 10 pursuit =5 }
+		drylands =  { damage = 10 pursuit =5 }
+		mountains = { damage = 14 }
+		hills = { damage = 8 toughness = 4 pursuit = 2}
```

Comment:
Hills add toughness, mountains don't. It's weird.
Also, the [Balance Spreadsheet](https://docs.google.com/spreadsheets/d/18zS-FAte6nsejx7rpzSmVuioRXps3iWSvSkx_xRJSWI/edit#gid=583000646&range=A142:A145) is outdated now together with the unit's cost.